libs/editor/example/src/main/java/org/wordpress/android/editor/example/EditorExampleActivity.java

```diff
@@ -252,7 +252,7 @@ public void run() {
                     while (count < 1.1) {
                         sleep(500);
 
-                        ((EditorMediaUploadListener) mEditorFragment).onMediaUploadProgress(mediaId, count);
+                        ((EditorMediaUploadListener) mEditorFragment).onMediaUploadProgress(mediaId, false, count);
```

Comment:
Doesn't look like the `false` param should be here anymore (`onMediaUploadProgress` wasn't changed) - also in two more places in this class.